channel/item.go

```diff
@@ -468,6 +468,63 @@ func (v Item) bytes(shortSlot bool) []byte {
 	return p
 }
 
+func (v Item) storageBytes() []byte {
```

Comment:
This function looks very similar to https://github.com/Hucaru/Valhalla/blob/master/channel/item.go#L393 I think the item `bytes()` should be modified to support both methods of encoding as it's preferable to not create multiple "serialize" functions for types if we can avoid it.